src/textacy/extract/triples.py

```diff
@@ -305,15 +304,105 @@ def expand_noun(tok: Token) -> list[Token]:
         child
         for tc in tok_and_conjuncts
         for child in tc.children
-        # TODO: why doesn't compound import from spacy.symbols?
```

Comment:
just wondering why this line was deleted? it's a comment, for me! :)

Reply:
i thought it was my comment lol